doc/api/modules.md

```diff
@@ -235,7 +235,7 @@ the `require.resolve()` function.
 Putting together all of the above, here is the high-level algorithm
 in pseudocode of what `require()` does:
 
-<pre>
```

Comment:
Why was this changed?

Reply:
This is covered in the commit message prior to this change being introduced: [490b02a](https://github.com/nodejs/node/pull/52883/commits/490b02ac554a35b9cccb476255df2dfa29c81bf0). What should I change in there to better articulate the issue?